openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -150,6 +150,29 @@ function download_ingress {
   done
 }
 
+function download_eventing_tls_testing_resources {
+
+  files=("$@")
+  echo "Files: ${files[*]}"
+
+  component_dir="$root/hack/lib/certmanager_resources"
+  target_dir="${component_dir}"
+
+  branch=$(metadata.get dependencies.eventing_artifacts_branch)
+  for ((i = 0; i < ${#files[@]}; i++)); do
+    index=$(( i+1 ))
+    file="${files[$i]}"
+    target_file="$target_dir/$file"
+
+    url="https://raw.githubusercontent.com/openshift-knative/eventing/${branch}/openshift/tls/issuers/$file"
+    echo "Downloading file from ${url}"
+    wget --no-check-certificate "$url" -O "$target_file"
```

Comment:
Is there a reason why we need to manually download this, instead of having the files in git?